docs/scos/dev/feature-integration-guides/202200.0/promotions-and-discounts-feature-integration.md

```diff
@@ -0,0 +1,851 @@
+---
+title: Promotions & Discounts feature integration
+description: Learn how to integrate the Promotions & Discounts feature into a Spryker project.
+template: feature-integration-guide-template
+---
+
+
+{% info_block errorBox %}
+
+The following feature integration guide expects the basic feature to be in place.
+
+The current sub-feature integration guide adds discount prioritization functionality..
+
+{% endinfo_block %}
+
+---
+
+## Install feature core
```

Comment:
```suggestion
## Install sub-feature core
```

Reply:
please check spelling everyewhere
Discount priorities is not a `feature`

Reply:
in this case we need opposite case
`sub-feature` and `prioritization` cases should be removed